core/linux-aarch64/config

```diff
@@ -9979,27 +9849,24 @@ CONFIG_NVDIMM_KEYS=y
 CONFIG_DAX=y
 CONFIG_NVMEM=y
 CONFIG_NVMEM_SYSFS=y
-# CONFIG_NVMEM_IMX_IIM is not set
-CONFIG_NVMEM_IMX_OCOTP=y
-CONFIG_NVMEM_IMX_OCOTP_SCU=y
-CONFIG_MTK_EFUSE=y
-CONFIG_QCOM_QFPROM=y
+CONFIG_NVMEM_APPLE_EFUSES=y
+# CONFIG_NVMEM_MESON_EFUSE is not set
+# CONFIG_NVMEM_MESON_MX_EFUSE is not set
+# CONFIG_NVMEM_MTK_EFUSE is not set
+# CONFIG_NVMEM_QCOM_QFPROM is not set
+CONFIG_NVMEM_RMEM=m
+# CONFIG_NVMEM_ROCKCHIP_EFUSE is not set
```

Comment:
I think this symbol was called `CONFIG_ROCKCHIP_EFUSE`  before and marked as =y

Reply:
Thanks, fixed in e12d5b1.